p2p/server.py

```diff
@@ -166,10 +167,13 @@ def __init__(self,
 
     async def receive_handshake(
             self, reader: asyncio.StreamReader, writer: asyncio.StreamWriter) -> None:
+        expected_exceptions = (
```

Comment:
Maybe move these to a top level constant?

Reply:
I'd rather keep them here as they're only expected when calling `_receive_handshake()`, and are not used anywhere else. Happy to move if you feel strong about that, though

Reply:
Motivation is:

- cutting out the overhead of building this tuple every time a handshake occurs.
- maybe increasing readability a bit by shortening function body.

I don't feel strongly as both of these are minor improvements.